.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+// These are GitHub Codespaces specific settings.
+{
+  "name": "Wasp Todo App Tutorial",
+  "customizations": {
+    "vscode": {
+      "extensions": ["wasp-lang.wasp", "PKief.material-icon-theme"],
+      "settings": {
+        "workbench.iconTheme": "material-icon-theme"
+      }
+    }
+  },
+  "containerEnv": {
+    "WASP_TELEMETRY_CONTEXT": "codespaces"
```

Comment:
Good one, we actually use "codespaces" in our analytics! So this will pick it up correctly.